src/Sql/Sql.Management.Sdk/Generated/Models/ChangeLongTermRetentionBackupAccessTierParameters.cs

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// Code generated by Microsoft (R) AutoRest Code Generator.
+// Changes may cause incorrect behavior and will be lost if the code is regenerated.
+
+namespace Microsoft.Azure.Management.Sql.Models
+{
+    using System.Linq;
+
+    /// <summary>
+    /// Contains the information necessary to change long term retention backup
+    /// access tier and related operation mode.
+    /// </summary>
+    public partial class ChangeLongTermRetentionBackupAccessTierParameters
```

Comment:
Do you know why this is added now? I thought we were trying to avoid adding it and eventually retire it